nikola/plugins/compile/pandoc.py

```diff
@@ -49,11 +50,27 @@ def set_site(self, site):
         self.config_dependencies = [str(site.config['PANDOC_OPTIONS'])]
         super().set_site(site)
 
+    def _get_pandoc_options(self, source: str) -> List[str]:
+        config_options: Union[List[str], Dict[str, List[str]]] = self.site.config['PANDOC_OPTIONS']
```

Comment:
Variable type annotations are not allowed in Python 3.5 (which we still support).

Reply:
Probably true. I know Python 3.5's type support is rudimentary. The better compromise make be make it a str. I'll check. I hate throwing away type information that helps people understand what's exactly going on.

Reply:
Oh... If indeed it doesn't work in Python 3.5, it is a weakness in the test that shows that it passes. It may be better to have the test that simply import the module even if it ain't checking for anything.

Reply:
Added test_importing_plugin_compile_pandoc following the example of test_importing_plugin_task_galleries. See if it is good enough.

Reply:
Fixed if test is passing now.